src/main/java/com/sap/oss/phosphor/fosstars/model/feature/oss/ArtifactVersionFeature.java

```diff
@@ -0,0 +1,33 @@
+package com.sap.oss.phosphor.fosstars.model.feature.oss;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.feature.AbstractFeature;
+import com.sap.oss.phosphor.fosstars.model.value.ArtifactVersionValue;
+
+/**
+ * This feature contains the version of the artifact for which the security rating is calculated.
+ */
+public class ArtifactVersionFeature extends AbstractFeature<String> {
```

Comment:
I think this feature should hold `ArtifactVersion` instead of a generic string.

```suggestion
public class ArtifactVersionFeature extends AbstractFeature<ArtifactVersion> {
```

Reply:
Same as for `ArtifactVersionValue`. The `ArtifactVersionFeature` is only the version as a string and is used to provide the version of the artifact for which the security rating is calculated.
The `ArtifactVersion` is the version string together with the release date (which we do not need for the used version).